xurls.go

```diff
@@ -81,6 +81,9 @@ var SchemesUnofficial = []string{
 	`zoomus`,        // Zoom (mobile)
 }
 
+var strictRe = regexp.MustCompile(strictExp())
+var relaxedRe = regexp.MustCompile(relaxedExp())
```

Comment:
Hmm. I specifically did not want the package to always compile the regexes at init time. If xurls ends up getting linked into a binary, but the regular expressions are never used (i.e. there are no calls to Strict or Relaxed), compiling the regexes will just burn CPU cycles at init time.

If you want to do this, I'd use a `sync.Once` so that they are compiled and stored globally on the first call to either Strict or Relaxed.

Reply:
Done.